packages/edit-post/src/components/sidebar/settings-header/index.js

```diff
@@ -27,66 +33,68 @@ const SettingsHeader = ( { sidebarName } ) => {
 		};
 	}, [] );
 
-	const [ documentAriaLabel, documentActiveClass ] =
+	const [ documentAriaLabel ] =
 		sidebarName === 'edit-post/document'
 			? // translators: ARIA label for the Document sidebar tab, selected. %s: Document label.
 			  [ sprintf( __( '%s (selected)' ), documentLabel ), 'is-active' ]
 			: [ documentLabel, '' ];
 
-	const [ blockAriaLabel, blockActiveClass ] =
+	const [ blockAriaLabel ] =
 		sidebarName === 'edit-post/block'
 			? // translators: ARIA label for the Block Settings Sidebar tab, selected.
 			  [ __( 'Block (selected)' ), 'is-active' ]
 			: // translators: ARIA label for the Block Settings Sidebar tab, not selected.
 			  [ __( 'Block' ), '' ];
 
-	const [ templateAriaLabel, templateActiveClass ] =
+	const [ templateAriaLabel ] =
 		sidebarName === 'edit-post/document'
 			? [ __( 'Template (selected)' ), 'is-active' ]
 			: [ __( 'Template' ), '' ];
 
-	/* Use a list so screen readers will announce how many tabs there are. */
 	return (
-		<ul>
-			{ ! isTemplateMode && (
-				<li>
-					<Button
-						onClick={ openDocumentSettings }
-						className={ `edit-post-sidebar__panel-tab ${ documentActiveClass }` }
-						aria-label={ documentAriaLabel }
-						data-label={ documentLabel }
-					>
-						{ documentLabel }
-					</Button>
-				</li>
-			) }
-			{ isTemplateMode && (
-				<li>
-					<Button
-						onClick={ openDocumentSettings }
-						className={ `edit-post-sidebar__panel-tab ${ templateActiveClass }` }
-						aria-label={ templateAriaLabel }
-						data-label={ __( 'Template' ) }
-					>
-						{ __( 'Template' ) }
-					</Button>
-				</li>
-			) }
-			<li>
-				<Button
-					onClick={ openBlockSettings }
-					className={ `edit-post-sidebar__panel-tab ${ blockActiveClass }` }
-					aria-label={ blockAriaLabel }
-					// translators: Data label for the Block Settings Sidebar tab.
-					data-label={ __( 'Block' ) }
+		<>
+			<Tabs.TabList>
+				<Tabs.Tab
+					id={ 'edit-post/document' }
+					render={
+						isTemplateMode ? (
+							<Button
```

Comment:
Is it necessary to `render` those `Tabs` as `Button`s?

Reply:
Good point. It is at the moment, but that's only because `Tabs.Tab` (and the other subcomponents) doesn't currently accept and forward all of the props being applied here. That's something we'd [already talked about improving](https://github.com/WordPress/gutenberg/pull/53960#pullrequestreview-1661817575) but I haven't done you. I'll get a separate PR together for that so this `render` prop is no longer necessary 👍

Reply:
PR to incorporate all HTML element props into the subcomponents: #55860

I've tested locally and it eliminates the need for these `render` props, so once that's merged, I can rebase and update this PR.

Reply:
With that PR merged, and this one rebased, I've removed the `render`s so the props can pass down normally ✅